.github/workflows/build-test-upload.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build, test and upload
+on:
+  pull_request:
+    types: [ 'closed' ]
+  workflow_dispatch:
+
+env:
+  FILES_DIR: roles/oneagent/files
```

Comment:
duplicated in [build-and-test.yaml](https://github.com/Dynatrace/Dynatrace-OneAgent-Ansible/pull/37/files#diff-a551b322da0f9fe92abd1ec41e43c58dc5d138c0000430395e66f0c581387cd2)